config/config.yml

```diff
@@ -118,7 +118,11 @@ arisa:
         MC-128302: duplicate-of-mc-128302
         MCL-5638: duplicate-of-mcl-5638
       privateMessage: duplicate-private
-      preventMessageTag: ARISA_NO_DUPLICATE_MESSAGE
+      preventMessageTags:
+        - ARISA_NO_DUPLICATE_MESSAGE
+        - ARISA_NO_DUPE_MESSAGE
+        - ARISA_NO_DUPLICATE_MSG
+        - ARISA_NO_DUPE_MSG
```

Comment:
I am not sure if it is really a good idea to have multiple alternatives here.
Keeping `ARISA_NO_DUPLICATE_MESSAGE` for legacy reasons is fine, but I think only one additional short tag (e.g. `ARISA_NO_DUPE_MSG`) should be used then. Otherwise it might cause confusion, what the "right" tag is